src/System.Windows.Forms/tests/AccessibilityTests/CusAccNameTesting.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Windows.Forms;
+
+namespace AccessibilityTests
+{
+    public partial class CusAccNameTesting : Form
```

Comment:
We don't have any constraints on the length of the type, so let's give it a clearer and more meaningful name, for example:
```suggestion
    public partial class CustomAccessiblePropertiesForm  : Form
```

Reply:
Don't forger rename files too 😊